### PR TITLE
Fixes for `simple-nft-module`

### PR DESCRIPTION
**Pull request by bkolad**
# Description
This PR introduces the following changes:
1. Adds `nft_path` to `GenesisPaths` so the nft genesis is read from a file.
2. Adds cli support in the `simple-nft-module`
3. Imports `simple-nft-module` in `demo-stf/Cargo.toml` (our customer is using it) 

## Linked Issues
- Fixes # (issue, if applicable)
- Related to #957

## Testing
Ci passes.

## Docs
No doc changes.

### CODE REVIEW

File: examples/simple-nft-module/src/tests.rs [hunk 1]
```diff
@@ -0,0 +1,27 @@
+use sov_modules_api::default_context::DefaultContext;
```
Comment (citizen-stig) [inline examples/simple-nft-module/src/tests.rs:1]: Could we please have tests to be not inside `src`, but in the module root?

File: module-system/module-implementations/sov-nft-module/src/tests.rs [hunk 2]
```diff
@@ -0,0 +1,14 @@
+use crate::NonFungibleTokenConfig;
```
Comment (citizen-stig) [inline module-system/module-implementations/sov-nft-module/src/tests.rs:1]: Same for tests location

**Review by vlopes11 — APPROVED** [review]
LGTM!

File: examples/simple-nft-module/src/tests.rs (hunk 1 above)
Comment (bkolad) [reply examples/simple-nft-module/src/tests.rs:1]: We had a long discussion offline. In my opinion this should be solved as a separate issue. @citizen-stig  Let me know if you are ok with that and I will create the issue.

Comment (bkolad) [reply examples/simple-nft-module/src/tests.rs:1]: https://github.com/Sovereign-Labs/sovereign-sdk/issues/993

**Pull request merged**